docs/src/site/apt/index.apt.vm

```diff
@@ -118,6 +118,15 @@ mvn scm-publish:publish-scm
 
 * History
 
+** 46
+
+  A <<GitHub ribbon>> has been added to the generated site to provide a direct link to the project's GitHub repository.
+
+  A new configuration property <<<maven.site.gitHubProjectId>>> has been introduced.
+  By default, this property is set to <<<$\{project.artifactId\}>>>, is used to resolve the <<GitHub>> repository URL for the ribbon link.
```

Comment:
1. for multi-module projects, a fixed value has to be added to the build root (can I add a commit to the PR?)
2. what are instructions when ribbon has to be disabled, like on maven-site or doxia-site? or do we really want the ribbon also on main sites?